test/conftest.py

```diff
@@ -0,0 +1,193 @@
+import pytest
```

Comment:
`conftest.py` is a pytest specific thing, so if we want to make it more general I should probably move it to a different file. Does anyone not use pytest, I don't know of anyone using unittest as the testrunner, rather than the framework (nose used to be a thing, but it's not been maintained for years)?

Reply:
> Does anyone not use pytest,

I sometimes use `unittest` as a runner for a few corner-cases where I can't wield pytest in the way I need to (most probably my fault not understanding how to do it). I also sometimes use the unittest runner from my IDE (Eclipse PyDev), but I just checked now and it can also be adjusted to use pytest underneath.

Anyway, more than my personal usage, what would concern me most would be older systems where NGAS is still installed, where adding more dependencies can result in more troubles to solve for system administrators. `unittest` on the other hand comes with the all python installations, and as such doesn't represent an extra burden.

As I said earlier the loss in flexibility is not too big, and as you point out pytest is fairly universal, so let's roll with this. *If* in the future we find we cannot use pytest in a system for one reason or another we can study how to disentangle this.